src/client.js

```diff
@@ -163,6 +163,13 @@ assign(Client.prototype, {
     return sql;
   },
 
+  postProcessResponse(resp, method) {
```

Comment:
here `method` is `undefined`. it's not passed here https://github.com/tgriesser/knex/pull/2261/files#diff-b27035181a616c40188ad22a32280732R136

Reply:
I actually decided in the last moment when implementing this, that I don't want to pass `method` to that handler, since it would expose too much internal implementation details of `knex`to end user. If something extra is passed it can be specified and added later (I was planning to use that to allow people to distinct between normal and raw responses).

Reply:
I'll remove this reminder of old initial implementation before merging.

Reply:
Makes sense, and it probably isn't necessary to know which method generated some data that needs post-processing

Reply:
> I was planning to use that to allow people to distinct between normal and raw responses

Precisely what I was looking for from this.